tests/atoms/gaia_apps.js

```diff
@@ -181,20 +181,26 @@ var GaiaApps = {
     var apps = window.wrappedJSObject.applications ||
                 window.wrappedJSObject.Applications;
     var app = apps.getByManifestURL(manifestURL);
-    var appName, launchPath;
+    var launchPath;
 
-    if (entryPoint) {
-      if (app.manifest.entry_points[entryPoint]) {
-        appName = app.manifest.entry_points[entryPoint].name;
-        launchPath = app.manifest.entry_points[entryPoint].launch_path;
-      } else {
-        app = null;
-      }
+    if (app != null) {
+      GaiaApps._formatApp(app, manifestURL, entryPoint, callback, launchPath);
```

Comment:
Yeah, I did in that outdated comment: https://github.com/mozilla-b2g/gaia/pull/32719#discussion_r43626487

Having 1 function that is used once helps to reduce the [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) of the code, and helps to add more context.